### ISSUE TITLE
Video streams are not loading

### ISSUE THREAD

**Issue by ethanjli** [sargassum-world/pslive#156]
For some reason, video streaming works when I load the instrument page immediately after starting the server. But then after I load the home page and leave it there for a few minutes, I'm no longer able to receive any video frames in the browser. There is also the same error message which gets emitted around the same time that the failure starts, though I'm not yet sure if it's related:

```
2022-12-03T05:18:55.637426589Z {"time":"2022-12-03T05:18:55.63722478Z","level":"WARN","prefix":"echo","file":"hub.go","line":"189","message":"removing subscription for /video-streams/external-stream/source.mjpeg?url=http%3A%2F%2Fpendelcam.kip.uni-heidelberg.de%2Fmjpg%2Fvideo.mjpg due to message receiver error: context canceled"}
```

**Comment by ethanjli** [comment]
I'm unable to reproduce this on the dev deployment (https://pslive-dev.cloud.syngnathus.sargassum.world/) on my development computer and on the prod deployment (https://live.sargassum.world), with my development database. It would be useful to try to determine whether adding instruments and cameras on my dev computer to match the configuration on the dev deployment enables me to reproduce the problem.

**Comment by ethanjli** [comment]
Another error message I'm seeing, which may or may not be related, is:
```
2022-12-13T22:38:02.759391619Z {"time":"2022-12-13T22:38:02.758445763Z","level":"ERROR","prefix":"echo","file":"cable.go","line":"53","message":"couldn't unmarshal client message: couldn't read websocket message: websocket: close 1005 (no status)"}
```

**Comment by ethanjli** [comment]
The prod deployment seems to also have failed, though I'm not sure whether the same error messages were emitted when it failed.

**Comment by ethanjli** [comment]
Something we can do to determine whether the issue is coming from WebSockets or elsewhere: in the broken dev deployment, try to load the MJPEG routes for proxied video streams, and see what happens

**Comment by ethanjli** [comment]
When I restart pslive on the cloud server and load https://pslive-dev.cloud.syngnathus.sargassum.world/video-streams/external-stream/stream.mjpeg?url=http://tenearth-3.prakashlab.d.live.fluitans.org:8000, it works fine; however, after I load the home page, it stops working. Similarly, if I open https://pslive-dev.cloud.syngnathus.sargassum.world/instruments/2 immediately after restarting pslive, it works fine, and refreshing it is fine too; when I open the homepage, everything is still fine.

In other words, I'm unable to reproduce the problem at all on the cloud deployment now.

**Comment by ethanjli** [comment]
The cloud deployments on both dev and prod seem to work fine (on v0.3.2) - in other words, the problem has gone away by itself and I can't fix it. The possible explanations I have are:

1. Something I fixed in #205 also fixed this behavior
2. There was something about the deployment environment (e.g. RAM constraints?) which transiently caused problems

For now I will close this issue; we can reopen it if the behavior appears again.
